pandas/tests/test_downstream.py

```diff
@@ -150,3 +150,11 @@ def test_missing_required_dependency():
     output = exc.value.stdout.decode()
     for name in ["numpy", "pytz", "dateutil"]:
         assert name in output
+
+
+def test_old_python_raises():
```

Comment:
we *could* move the import of just ``pandas.compat`` at the beginning of init and define PY35 there (and import here); would put the code more logically together.